app/jobs/calendar_importer/event_resolver.rb

```diff
@@ -180,10 +161,6 @@ def save_all_occurences
       unless event.update(attributes)
         notices << event.errors.full_messages.join(', ')
       end
-
-      if event.address_id.blank? && calendar.strategy == 'event'
```

Comment:
I think I remember you saying that events will no longer require an address.

(So can only be matched through partner)